src/Tribe/Main.php

```diff
@@ -249,8 +253,19 @@ public function load_assets() {
 				),
 			)
 		);
+	 }
+
+	/**
+	 * Load All localization data create by `asset.data`
+	 *
+	 * @since  TBD
+	 *
+	 * @return void
+	 */
+	public function load_localize_data() {
```

Comment:
Nitpicky and not a blocker, but should this be `load_localized_data`?

Reply:
That was how it was on RER, I would rather not change to avoid merge conflicts if possible

Reply:
That makes sense 👍